ExtendedDescription.md

```diff
@@ -0,0 +1,130 @@
+
+## Code of Conduct
+
+Nelson is home to people from all over the world who have a diverse set of skills, personalities, backgrounds and experiences. This diversity is wonderful, and we are committed to ensuring that the nelson.social community is a safe, enjoyable and thriving environment for the largest number possible of Nelson residents, their friends and family members.
+
+To that end, we will strive to make this a friendly, safe and welcoming environment for everyone regardless of qualities such as their gender identity or expression, sexual orientation, disability, mental illness, neuro(a)typicality, physical appearance, body size, age, race, nationality, ethnicity, socioeconomic status, family structure, spirituality, religion (or lack thereof), education, or other personal traits. We particularly celebrate diversity and do not tolerate bigotry or prejudice.
+
+As a community, we expect any participant to act according to the following ground rules. They ensure that everyone within the community is clear about the behaviour that is expected of them, and that newcomers know what kind of environment they can expect to find here.
+
+This is a living document, and the nelson.social team welcome your feedback. We will amend this document as necessary to ensure the safety of our users.
+
+### Guiding principle
+
+*If you wouldn't do it in Oso Negro, don't do it on nelson.social*
+
+### Handling conflict
+
+A diverse group of people has a diverse set of boundaries about what is OK. Sometimes one of us may make a mistake, and do something that inadvertently causes offence. When we observe this happen, we try to give the offender feedback quickly, clearly and constructively. We try to offer them the opportunity to maintain their dignity in the situation, to learn from and repair their mistake.
+
+This kind of learning and repair is how we strengthen our relationships and our community. However we will not tolerate anyone deliberately or repeatedly breaking these rules.
+
+### Consequences of Unacceptable Behavior
+
+Unacceptable behavior from any community member, including those with decision-making authority, will not be tolerated.
+
+Anyone asked to stop unacceptable behavior is expected to comply immediately.
+
+If a community member engages in unacceptable behavior, the community organizers may take any action they deem appropriate, up to and including a temporary ban or permanent expulsion from the community without warning.
+
+### Reporting Guidelines
+
+If you are subject to or witness unacceptable behavior, or have any other concerns, please [notify a community organizer](#team) by emailing [admin@nelson.social](mailto:admin@nelson.social) as soon as possible. We are here to help you. Your reports will be taken seriously, treated in confidence and will not be dismissed or argued with.
```

Comment:
"Notify a community organizer via direct message, or email admin@nelson.social"

Maybe that to make it less formal to always use email? It would be quicker for us as well to react to.

Reply:
Maybe we should also mention that people can "report" toots directly (on mobile and web app).

Reply:
Maybe there is a level 1 "please take a look at this, it offensive" and a level 2 "DM/email the team"?

Since mastodon has moderation/reporting built in we may as well use it.

Reply:
I thought about that but don't know mastodon well enough yet to be able to describe it. It would be great if you want to try and add some detail here.

Reply:
```suggestion
If you are subject to or witness unacceptable behavior, please use the "Report user" functionality of Mastodon. If the situation is beyond unacceptable behavior, please notify any of the [Team](#team) via direct message. We are here to help you. Your reports will be taken seriously, treated in confidence and will not be dismissed or argued with.
```